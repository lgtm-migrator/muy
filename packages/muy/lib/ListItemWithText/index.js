import React from "react"
import ListItem from "@material-ui/core/ListItem"
import ListItemText from "@material-ui/core/ListItemText"

const ListItemWithText = props => (
  <ListItem>
    <ListItemText {...props} />
  </ListItem>
)

export default ListItemWithText
