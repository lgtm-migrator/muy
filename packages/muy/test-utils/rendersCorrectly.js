import renderer from "react-test-renderer"

const rendersCorrectly = component => {
  const tree = renderer.create(component).toJSON()
  return expect(tree).toMatchSnapshot()
}

export default rendersCorrectly
