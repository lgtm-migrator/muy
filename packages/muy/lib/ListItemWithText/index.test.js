import React from "react"
import rendersCorrectly from "../../test-utils/rendersCorrectly"
import ListItemWithText from "."

describe("<ListItemWithText/> component", () => {
  it("should render correctly with no props", () => {
    rendersCorrectly(<ListItemWithText />)
  })
})
