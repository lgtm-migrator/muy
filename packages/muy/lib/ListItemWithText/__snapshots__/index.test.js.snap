// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListItemWithText/> component should render correctly with no props 1`] = `
<li
  className="MuiListItem-root MuiListItem-gutters"
  disabled={false}
>
  <div
    className="MuiListItemText-root"
  />
</li>
`;
